startup.m

```diff
@@ -0,0 +1,6 @@
+% startup.m
+setPath ;
+setenv('DJ_HOST', 'at-database.ad.bcm.edu');
+setenv('DJ_USER','spatel');
+setenv('DJ_PASS','spatel');
```

Comment:
argh!, removed the startup from the location where it was stored, moved the credentials to matlab's default startup on the machine, also changed password